src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/components/SampleFiltering/index.tsx

```diff
@@ -65,12 +70,14 @@ export function SampleFiltering() {
         </StyledFilterGroup>
         <StyledFilterGroup>
           <StyledFilterGroupName>Collection Date</StyledFilterGroupName>
-          <StyledInputDropdown
-            disabled={false}
-            label="Dummy placeholder"
-            onClick={noop}
-            sdsStage="userInput"
-            sdsStyle="square"
+          <CollectionDateFilter
+            fieldKeyEnd="collectionDateEnd"
+            fieldKeyStart="collectionDateStart"
+            updateDateFilter={noop}
```

Comment:
why is this noop?

Reply:
Because the filter does not actually do anything yet. As I mentioned in the PR notes, I still need to have the filter impact the resulting tree. That is in the next PR

Reply:
ok got it! thanks